packages/sync-actions/test/product-sync-prices.spec.js

```diff
@@ -169,6 +169,101 @@ describe('Actions', () => {
     expect(actions).toEqual([])
   })
 
+  test('should generate PriceCustom actions before changePrice action', () => {
```

Comment:
Perfect and thanks for adding this test 👍 This way we can be sure that further changes won't break it 😎